robot_controllers/include/robot_controllers/trajectory.h

```diff
@@ -108,11 +110,28 @@ inline bool trajectoryFromMsg(const trajectory_msgs::JointTrajectory& message,
     TrajectoryPoint point;
     for (size_t j = 0; j < joints.size(); ++j)
     {
-      point.q.push_back(message.points[p].positions[mapping[j]]);
-      if (message.points[p].velocities.size() == message.points[p].positions.size())
+      bool has_positions = true;
+      // If you get velocities but not positions
+      if (message.points[p].positions.size() < message.points[p].velocities.size())
```

Comment:
If `message.points[p].positions.size()` is not 0 but still less than `message.points[p].velocities.size()`, we'll still be using velocity only?